.github/workflows/lintingci.yml

```diff
@@ -58,3 +58,21 @@ jobs:
       - name: Run TFLint
         working-directory: forest-node/terraform
         run: tflint -f compact
+
+  fmt_terraform:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Setup terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+           path: ~/.tflint.d/plugins
+           key: tflint-${{ hashFiles('.tflint.hcl') }}
```

Comment:
What is this exactly, and why do we need this?

Reply:
More specifically, why do we need to override the defaults, whatever they are?

Reply:
Oops, that's a copy and paste mistake.